### PR TITLE
fix(islandis-login): fixing all island.is login validations relying on islandis-login

### PR DESCRIPTION
**Pull request by arnorhs**
## Checklist:

- [x] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] Formatting passes locally with my changes
- [x] I have rebased against main before asking for a review

### CODE REVIEW

**Review by pshomov — APPROVED** [review]
lgtm

File: apps/air-discount-scheme/backend/src/app/modules/flight/flight.controller.ts [hunk 1]
```diff
@@ -25,7 +25,6 @@ import {
   ApiResponse,
 } from '@nestjs/swagger'
 
-import IslandisLogin, { VerifyResult } from 'islandis-login'
```
Comment (eirikurn) [inline apps/air-discount-scheme/backend/src/app/modules/flight/flight.controller.ts:28]: Unused?

**Review by eirikurn — APPROVED** [review]

File: apps/air-discount-scheme/backend/src/app/modules/flight/flight.controller.ts (hunk 1 above)
Comment (eirikurn) [reply apps/air-discount-scheme/backend/src/app/modules/flight/flight.controller.ts:28]: Yup

Comment (arnorhs) [reply apps/air-discount-scheme/backend/src/app/modules/flight/flight.controller.ts:28]: apparently, yes :)

**Pull request merged**